js_modules/dagit/packages/core/src/runs/MetadataEntry.tsx

```diff
@@ -229,47 +230,31 @@ const MetadataEntryModalAction: React.FunctionComponent<{
   content: () => React.ReactNode;
   copyContent: () => string;
 }> = (props) => {
-  const [isExpanded, setExpanded] = React.useState(false);
+  const [open, setOpen] = React.useState(false);
   return (
     <>
-      <MetadataEntryAction onClick={() => setExpanded(true)}>{props.children}</MetadataEntryAction>
-      {isExpanded && (
-        <Dialog
-          icon="info-sign"
-          usePortal={true}
-          style={{width: 'auto', minWidth: 400, maxWidth: '80vw'}}
-          title={props.label}
-          onClose={() => setExpanded(false)}
-          isOpen={true}
-        >
-          <MetadataEntryModalContent>{props.content()}</MetadataEntryModalContent>
-          <div className={Classes.DIALOG_FOOTER}>
-            <div className={Classes.DIALOG_FOOTER_ACTIONS}>
-              <Button onClick={(e: React.MouseEvent) => copyValue(e, props.copyContent())}>
-                Copy
-              </Button>
-              <Button intent="primary" autoFocus={true} onClick={() => setExpanded(false)}>
-                Close
-              </Button>
-            </div>
-          </div>
-        </Dialog>
-      )}
+      <MetadataEntryAction onClick={() => setOpen(true)}>{props.children}</MetadataEntryAction>
+      <DialogWIP
```

Comment:
This one is a pretty significant change since we're rendering dialogs into the DOM all the time now. Should be fine but if there are a lot of `MetadataEntryModalAction` on the page this might be a lot of stuff getting added to the DOM tree that wasn't there before

Reply:
The contents of the Dialog shouldn't be rendered up front, since `lazy` is true by default in Blueprint. (I believe this is the case, will double check.)

Reply:
Yeah, it seems fine.